server/app/views/api/ApiDocsView.java

```diff
@@ -189,11 +188,19 @@ private DivTag apiResponseSampleDiv(ProgramDefinition programDefinition) {
     String fullJsonResponsePreview = programJsonSampler.getSampleJson(programDefinition);
     String fullJsonResponsePreviewPretty = asPrettyJsonString(fullJsonResponsePreview);
 
-    apiResponseSampleDiv.with(
-        pre(code(fullJsonResponsePreviewPretty))
-            .withStyle("max-width: 100ch;")
-            .withClasses(
-                "m-4", "p-2", "rounded-lg", "bg-slate-200", "break-words", "whitespace-pre-wrap"));
+    apiResponseSampleDiv
+        .attr("role", "complementary")
+        .attr("aria-label", "API response preview in JSON format")
```

Comment:
suggestion (non-blocking):  I think it's preferable to use a `label` tag when the label is visible text.  [When to use label and aria-label](https://accessibleweb.com/question-answer/what-are-the-differences-between-label-aria-label-aria-labelledby-and-aria-describedby/#:~:text=Typical%20examples%20of%20graphic%20labels,interacting%20with%20the%20page%20visually.).  Could we do that here?